### PR TITLE
[HW-957] Add remote start/stop on OCPP2.0.1.

### PR DESCRIPTION
**Pull request by rustik666**
[HW-957] Add remote start/stop on OCPP2.0.1.

### CODE REVIEW

File: src/lib/ChargeStation/eventHandlers/ocpp-20/handle-request-stop-transaction.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { ChargeStationEventHandler } from 'lib/ChargeStation/eventHandlers';
+import {RequestStopTransactionRequest} from "schemas/ocpp/2.0/RequestStopTransactionRequest";
+import {RequestStopTransactionResponse} from "schemas/ocpp/2.0/RequestStopTransactionResponse";
+
+const handleRequestStartTransaction: ChargeStationEventHandler<RequestStopTransactionRequest> =
```
Comment (nick-jones) [inline src/lib/ChargeStation/eventHandlers/ocpp-20/handle-request-stop-transaction.ts:5]: `handleRequestStartTransaction` -> `handleRequestStopTransaction`

**Review by nick-jones — APPROVED** [review]
Quite a few formatting discrepencies, up to you if you want to fix them - we should probably add prettier here at some point :+1:

File: src/lib/ChargeStation/eventHandlers/ocpp-20/send-authorize.ts [hunk 2]
```diff
@@ -11,10 +9,10 @@ const sendAuthorize: ChargeStationEventHandler<
...
   await sleep(1000);
 
-  await chargepoint.writeCall(
-    'Authorize',
-    { idToken: { idToken: session.options.uid, type: 'ISO14443' } },
-    session
+  chargepoint.writeCall(
+      'Authorize',
+      {idToken: {idToken: session.options.uid, type: 'ISO14443'}},
+      session
```
Comment (x-cray) [inline src/lib/ChargeStation/eventHandlers/ocpp-20/send-authorize.ts:15]: Check your formatter, it inserts 4 spaces, and removes padding from object literal.

File: src/lib/ChargeStation/eventHandlers/ocpp-20/send-start-transaction.ts [hunk 3]
```diff
@@ -29,34 +29,35 @@ const sendStartTransaction: ChargeStationEventHandler = async ({
...
-      timestamp: startTime,
-      triggerReason: 'ChargingStateChanged',
-      seqNo: session.seqNo,
-      transactionInfo: {
-        transactionId: session.transactionId,
-        chargingState: 'SuspendedEVSE',
-      },
-      meterValue: [
+    chargepoint.writeCall(
+        'TransactionEvent',
```
Comment (x-cray) [inline src/lib/ChargeStation/eventHandlers/ocpp-20/send-start-transaction.ts:33]: Same here, formatting goes against the codebase

**Review by x-cray — APPROVED** [review]

**Pull request merged**